### PR TITLE
OPIOID-103 Implement demographic questions

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: themes/ohcoe/static/js/scripts.js [hunk 1]
```diff
@@ -68,4 +69,30 @@ document.addEventListener('DOMContentLoaded', function() {
     if (scoreContainer && domainScoreGrowth) {
         scoreContainer.innerHTML = 'You have this much growth: <span id="score-container">' + domainScoreGrowth + '%</span>';
     }
+
+    /* Demographic Questions */
+    document.querySelectorAll('.demographic-questions input[type="text"]').forEach(function(el){
+        el.addEventListener('focus', function(e){
```
Comment (sdreher) [inline themes/ohcoe/static/js/scripts.js]: Some linting things here. Is `eslint` turned on?

File: themes/ohcoe/static/js/scripts.js [hunk 2]
```diff
@@ -68,4 +69,30 @@ document.addEventListener('DOMContentLoaded', function() {
...
+            var radioInput = e.target.parentNode.querySelector('input[type="radio"]');
+            radioInput.setAttribute('checked', 'true');
+        })
+        el.addEventListener('blur', function(e){
+            var radioInput = e.target.parentNode.querySelector('input[type="radio"]');
+            radioInput.setAttribute('value', e.target.value);
+        })
+    });
+    document.querySelector('#demographic-info').addEventListener('submit', function(e){
+        e.preventDefault();
```
Comment (sdreher) [inline themes/ohcoe/static/js/scripts.js]: Any particular reason to query for this demographic information? Is the experience tuned based on the audience?

Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: Nope, it was requested by the clients.  The term 'demographic' is used by them, which initially raised my concern, but the demographics here are related to their role and speciality as dentists. Its not asking about categories that one would normally think of as demographic.

As part of the back and forth with the client, we asked that there be a 'Prefer not to answer' option, which is set as the default choice.

File: themes/ohcoe/static/js/scripts.js (hunk 1 above)
Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: Nope not yet.  Given the extent of the js here, I can add that as a commit to this PR.

Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: I thought I'd be clever and not use jQuery for this site but:

**Pull request merged**